lib.sh

```diff
@@ -91,6 +91,7 @@ DISK="$GITHUB_REPO/disk"
 NETWORK="$GITHUB_REPO/network"
 VAGRANT_DIR="$GITHUB_REPO/vagrant"
 NOT_SUPPORTED="$GITHUB_REPO/not-supported"
+DAT_FILES="$GITHUB_REPO/dat"
```

Comment:
rename the folder to `geoblockdat` please

Reply:
done

Reply:
Uh, I've only renamed the variable. Do this later.

Reply:
done